server/src/test/java/org/opensearch/threadpool/ScalingThreadPoolTests.java

```diff
@@ -134,6 +134,7 @@ private int expectedSize(final String threadPoolName, final int numberOfProcesso
         sizes.put(ThreadPool.Names.FETCH_SHARD_STORE, ThreadPool::twiceAllocatedProcessors);
         sizes.put(ThreadPool.Names.TRANSLOG_TRANSFER, ThreadPool::halfAllocatedProcessorsMaxTen);
         sizes.put(ThreadPool.Names.TRANSLOG_SYNC, n -> 4 * n);
+        sizes.put(ThreadPool.Names.TRANSLOG_REMOTE_PURGE, ThreadPool::halfAllocatedProcessorsMaxFive);
```

Comment:
should we just have `REMOTE_PURGE` , which can be reused in segments deletion as well ?

Reply:
Incorporated